qemu/tests/rng_hotplug.py

```diff
@@ -76,20 +80,31 @@ def stop_rngd(vm):
                 raise exceptions.TestError(output)
             session.close()
 
+    def check_and_run_subtest(check_test):
+        """
+        Run subtest when it's not None
+        :param check_test: subtest name
+        """
+        if check_test:
```

Comment:
it's better to check the param out of the function
and suggest to update the function name